### PR TITLE
[BUGFIX][ISSUE-448] fix Undefined array key exceptions on PHP 8.0

### PR DESCRIPTION
**Pull request by Fanor51**
Like in the ISSUE described -> https://github.com/TYPO3-Headless/headless/issues/448.

I saw that the first exception was already fixed in master but the second error wasn´t yet.
Here my try to fix the issue, maybe it could help.

Best regards
Fabio Norbutat

### CODE REVIEW

File: Classes/Utility/FileUtility.php [hunk 1]
```diff
@@ -134,8 +134,10 @@ public function processFile(FileInterface $fileReference, array $dimensions = []
                 'height' => $this->getCroppedDimensionalProperty($fileReference, 'height', $cropVariant)
             ],
             'crop' => $crop,
-            'autoplay' => $fileReference->getProperty('autoplay'),
-            'extension' => $fileReference->getProperty('extension'),
+            'autoplay' => $fileReference->hasProperty('autoplay')
+                ? $fileReference->getProperty('autoplay') : 0,
+            'extension' => $fileReference->hasProperty('extension')
+                ? $fileReference->getProperty('extension') : 0,
```
Comment (twoldanski) [inline Classes/Utility/FileUtility.php:140]: extensions is not a int, it should return string|null

**Review by twoldanski — APPROVED** [review]

**Comment by Fanor51** [comment]
@twoldanski i am allready on them.

**Comment by Fanor51** [comment]
@twoldanski,
allright i figured out from where this mis-behavior was from. Take a look on my last commit please.

Thanks!

**Review by twoldanski — APPROVED** [review]

**Comment by twoldanski** [comment]
@Fanor51 thank you! looks fine! approved @lukaszuznanski merge please

**Pull request merged**